modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -629,22 +629,22 @@ protected void doInitialise() throws InitialisationException {
         boolean isPrimaryPollingInstance = clusterService.isPrimaryPollingInstance();
         if (primaryNodeOnly) {
           LOGGER
-              .debug("Message source '{}' on flow '{}' is being initialised, it is running on the primary node and it must only run on this node.",
+              .debug("Message source '{}' on flow '{}' running on the primary node is initializing. Note that this Message source must run on the primary node only.",
                      sourceModel.getName(), getLocation().getRootContainerName());
         } else {
           LOGGER
-              .debug("Message source '{}' on flow '{}' is being initialised, there are no cluster restrictions regarding which node must run the source. This node {} the primary node.",
+              .debug("Message source '{}' on flow '{}' is initializing. This {} the primary node of the cluster.",
```

Comment:
Minor typo. Please add "is" i.e.  "This {} is the primary node of the cluster."

Reply:
The {} placeholder will be transformed into either "is" or "is not" depending if it is the primary node or not.